app/models/user.rb

```diff
@@ -120,6 +120,9 @@ def suspend!
       raise 'user_already_suspended'
     end
     update!(suspended_at: Time.zone.now)
+    active_identities.each do |identity|
+      OutOfBandSessionAccessor.new(identity.session_uuid).destroy
+    end
```

Comment:
should we also update the `session_uuid` in each of these identities to be null?

Reply:
Sure - I can update it to be `rails_session_id` - but I'm not sure about nulling the session id's - I'll follow up in Slack and defer to you on that.